cookbooks/vectortile/files/default/html/robots.txt

```diff
@@ -0,0 +1,13 @@
+User-agent: *
+Disallow: /12/
+Disallow: /13/
+Disallow: /14/
+Disallow: /15/
+Disallow: /16/
+Disallow: /17/
+Disallow: /18/
+Disallow: /19/
+Disallow: /20/
+Disallow: /21/
+Disallow: /22/
+Disallow: /23/
```

Comment:
Is this right for vector tiles?

Reply:
Zoom levels 20+ don't exist for our raster or vector tiles. I assumed they were added to robots.txt because people were requesting them, and the same could happen with vector tiles.